source/common/tracing/BUILD

```diff
@@ -39,3 +36,17 @@ envoy_cc_library(
         "//source/common/runtime:uuid_util_lib",
     ],
 )
+
+envoy_cc_library(
+    name = "lightstep_tracer_lib",
+    srcs = [
```

Comment:
If it's OK with you I'd rather not wait for another 40 minute travis build for this.  I'll do single line unless buildifier overrules in the future.